### ISSUE TITLE
Dependency on zend-ldap

### ISSUE THREAD

**Issue by stijnhau** [stanislav-web/ZF2-PHP-WebSocket-Server-Factory#7]
PHP Fatal error:  Class 'Zend\Ldap\Ldif\Encoder' not found in /home/stijnhau/DropboxEmsys/workspace/Carhole/vendor/stanislav-web/zf2-websocket-server-factory/src/WebSockets/Service/WebsocketServer.php on line 768

**Comment by stanislav-web** [comment]
Zend\Ldap is not included any more in a default installation. As the ZendFramework-Team has splitted the zendframework into multiple components not all of them are needed any more. And Zend\Ldap is one of the "not needed" ones as it has a hard dependency on ext/ldap as you can't use it without the LDAP-extension of PHP. For details on that have a look at https://github.com/zendframework/zf2/issues/7569

You should therefore run composer require zendframework/zend-ldap from the base-directory of your project and you should be up and running. By now you get that as a suggestions after the installation.

zendframework/zendframework suggests installing zendframework/zend-ldap (zend-ldap component ~2.5.0, if you need LDAP features)

**Comment by stijnhau** [comment]
If that is the fox(Not tested it myself)
Then you should add the requirement of that module to you composer.json file.

**Comment by stanislav-web** [comment]
Sorry for late action. 
Already added . Thanks!

**Comment by stijnhau** [comment]
Yes now only a stable version is needed because most of us don't want to use the dev master.

**Comment by stanislav-web** [comment]
Ok, i 'll test dev to get stable
